docs/web-chat/users.mdx

```diff
@@ -173,3 +183,228 @@ When making a request to Helpshift with no `userAuthToken` or an invalid `userAu
+    })
+
+    Helpshift("login", {
+      identity_token: identityToken,
+      full_privacy_enabled: false // or true. Assumed false if not specified
+    });
+});
+
+// At some point in the future, when your user logs in:
+window.helpshiftConfig.userId = "<your-user-id>"
+window.helpshiftConfig.userEmail = "<your-user-email>"
+
+Helpshift("updateHelpshiftConfig")
+// ^ This will trigger the onLoginRequest event.
+```
+
+### Logging out your end-users {#logout-end-users}
+To log out your end-users, you can use the logout API.
+
+
+```js
+Helpshift("logout");
+```
+
+Note that after calling this API, Web Chat will clear any user specific data from the `helpshiftConfig`
+object, and reset the chat to an anonymous user. Depending on the value of `helpshiftConfig.clearAnonymousUserOnLogin`,
+this can either be the previous anonymous user or a new anonymous user.
+
+<Admonition type="info" title="Tip">
+
+For a better user experience, consider hiding the chat widget after the user logs out.
+
+</Admonition>
+
+### Optimizations
+
+<Admonition type="info" title="Note">
+
+Web Chat will run smoothly even without these optimizations. However, if you want to see
+faster load times for Web Chat, and reduced load on your infrastructure, you may choose to implement these
+optimizations.
+
+</Admonition>
+
+#### Send the identity_token in SSR requests {#send-identity-token-in-ssr}
+If your application uses SSR, you can send the `identity_token` as part of your page load,
+injecting it in the `onLoginRequest` callback directly.
+
+_Please ensure that you do not set this value in a global variable to minimize the security risks for your
+end-user._
+
+<Admonition type="info" title="Caveat">
+
+Helpshift fires the `onLoginRequest` event if we detect suspicious end-user activity. In such cases,
+your page will provide a stale ID token, which will result in an expired support session.
+
+A refresh of the page would allow their session to be restored.
+
+This is usually not an issue for most end-users, but in case you want instant support session recovery, a JWT endpoint
+would become necessary.
+
+</Admonition>
+
+#### Caching the identity_token {#caching-id-token}
+We highly discourage caching the `identity_token` on the client-side since it increases the potential
+attack surface for XSS. However, you may choose to cache it on the server-side, provided that the cache
+is invalidated before 24 hours elapse.
+
+### Troubleshooting and tips {#troubleshooting-and-tips}
+
+Login failure can be detected by listening to the [`onWebChatRuntimeError` event](/web-chat/api/#on-webchat-runtime-error).
+The possible reasons for login failure are cited in the [`LOGIN_FAILED` runtime errors](/web-chat/api/#login-failed).
+
+If your login is still failing, you can investigate these other potential reasons:
+1. Ensure that your JWT endpoint is generating tokens in the accepted format
+2. Ensure that your JWT endpoint is signing tokens with the shared secret
```

Comment:
Should we link to the KB article of the secret key generation? This article: https://helpshift.atlassian.net/wiki/spaces/~7120201150c255595f403cad6652223bcf8661/pages/4605083915/User+Hub+User+Data+Security